packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/RuntimeError/GroupedStackFrames.tsx

```diff
@@ -0,0 +1,59 @@
+import type { StackFramesGroup } from '../../helpers/group-stack-frames-by-framework'
```

Comment:
this file is not used anymore, we can delete the unused ones before forking so we dont need to delete twice later

Reply:
Delete unused ones in a separate PR please.